core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -226,6 +226,48 @@ oppia.controller('SettingsTab', [
     /********************************************
     * Methods relating to control buttons.
     ********************************************/
+    $scope.previewSummaryTile = function() {
+      alertsService.clearWarnings();
+      $modal.open({
+        templateUrl: 'modals/previewSummaryTile',
+        backdrop: true,
+        controller: [
+          '$scope', '$modalInstance', function($scope, $modalInstance) {
+            $scope.getExplorationTitle = function() {
+              return explorationTitleService.displayed;
+            };
+            $scope.getExplorationObjective = function() {
+              return explorationObjectiveService.displayed;
+            };
+            $scope.getExplorationCategory = function() {
+              return explorationCategoryService.displayed;
+            };
+            $scope.getThumbnailIconUrl = function() {
+              var category = explorationCategoryService.displayed;
+              if (GLOBALS.ALL_CATEGORIES.indexOf(category) === -1) {
+                category = 'Lightbulb';
+              }
+              return '/assets/images/subjects/' +
```

Comment:
Look at Splash.js -- use UrlInterpolationService.getStaticImageUrl. In production, assets are served from a different place.